client/my-sites/invites/invite-accept-logged-out/index.jsx

```diff
@@ -60,11 +60,40 @@ export default React.createClass( {
 		)
 	},
 
-	footerLink() {
+	subscribeUserByEmailOnly() {
+		this.setState( { submitting: true } );
+		acceptInvite(
+			this.props.invite,
+			( error ) => {
+				if ( error ) {
+					this.setState( { error } );
+				} else {
+					window.location = 'https://subscribe.wordpress.com?update=activate&email=' + encodeURIComponent( this.props.invite.meta.sent_to ) + '&key=' + this.props.invite.authKey;
```

Comment:
Why do we need to send the user to `subscribe.wordpress.com` if we're using the API to accept the invite?